go.mod

```diff
@@ -87,7 +85,8 @@ require (
 	github.com/golangci/plugin-module-register v0.1.1 // indirect
 	github.com/golangci/revgrep v0.5.3 // indirect
 	github.com/golangci/unconvert v0.0.0-20240309020433-c5143eacb3ed // indirect
-	github.com/google/go-cmp v0.6.0 // indirect
+	github.com/google/go-cmp v0.7.0 // indirect
+	github.com/google/go-github/v78 v78.0.0 // indirect
```

Comment:
Unfortunately this provider is on v67 and the upgrade has not been made to get us to v78 yet.

Reply:
@sheeeng We're [planning](https://github.com/integrations/terraform-provider-github/milestone/60) on releasing the provider on [v79](https://github.com/integrations/terraform-provider-github/pull/2898) shortly.  I'd love to get this change in that release.  Would you mind fixing the merge conflicts and CI issues so that we can get this a proper review ❤️